### PR TITLE
Submit scanned voucher after one second and fade out inputted text ov…

### PR DESCRIPTION
**Pull request by ob6160**
…er this time period

### CODE REVIEW

File: src/pages/Scan.vue [hunk 1]
```diff
@@ -62,7 +65,7 @@ import Store from '../store.js';
 import Profile from '../components/Profile.vue';
 import Queue from '../components/Queue.vue';
 
-const RESULT_TIMER = 2000;
+const RESULT_TIMER = 1000;
```
Comment (charlesstrange2) [inline src/pages/Scan.vue]: total nitpick: SCAN_TIMEOUT, or similar?

Comment (charlesstrange2) [inline src/pages/Scan.vue]: We're avoiding ES6 at the moment :cry: just in case :fear:

**Review by charlesstrange2 — DISMISSED** [review]
A trivial nitpick or two and we need a test script.

File: src/pages/Scan.vue (hunk 1 above)
Comment (ob6160) [reply src/pages/Scan.vue]: I think we should keep this the same name as the const in Tap.vue as we want to - at some point - share functionality between the two. We'll probably have to change it back then anyway.

**Comment by charlesstrange2** [comment]
Good point - there's a long grass card about combining tap and scan in one
thing.

File: src/pages/Scan.vue (hunk 1 above)
Comment (ob6160) [reply src/pages/Scan.vue]: Changed

**Review by gingerbreadpup — COMMENTED** [review]
Yep :shipit:

**Review by gingerbreadpup — APPROVED** [review]

**Pull request merged**